### ISSUE TITLE
API reference link gives this 

### ISSUE THREAD

**Issue by kjeel-dev** [equinor/omnia-timeseries-python#54]
API Reference link at the bottom of the README (https://api.equinor.com/docs/services/Timeseries-api-v1-7) gives page does not exist.
![image](https://github.com/equinor/omnia-timeseries-python/assets/77051914/3f1cc2b7-7e6e-4274-9486-d5df633cf90d)

**Comment by luizamfsantos** [comment]
I found this page in their website: https://api.equinor.com/api-details#api=Timeseries-api-v1-7&operation=writeData

**Comment by kjeel-dev** [comment]
Yes, thank you, I found that one too. I just think that the non-functioning link should be updated.
